src/annotations/items/AbstractAnnotationItem.h

```diff
@@ -42,7 +42,7 @@ class AbstractAnnotationItem : public QGraphicsWidget
 	QPainterPath shape() const override;
 	virtual bool intersects(const QRectF &rect) const;
 	virtual void addPoint(const QPointF &position, bool modified) = 0;
-	virtual void setPointAt(const QPointF &point, int index) = 0;
+    virtual void setPointAt(const QPointF &point, int index, bool keepAspectRatio = false) = 0;
```

Comment:
I'm personally not longer a fan of default values and I think it's not recommend to use them in virtual methods. I would suggest to either always specify the keepAspectRatio value or have two method with different parameters.